packages/serverpod_shared/lib/src/config.dart

```diff
@@ -114,6 +114,14 @@ class ServerConfig {
 
     return str;
   }
+
+  /// Get [Uri] from publicHost, port and publicScheme.
+  Uri toUri() {
+    if (publicScheme == 'http') {
```

Comment:
Writing code this way assumes that we will never add more schemes. It's better to test for both `http` and `https` and throw an exception if none of the schemes are available.

Otherwise, this can cause a hard to find bug in the future.